code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -114,7 +114,10 @@
 	if(stat == DEAD || (HAS_TRAIT(src, TRAIT_FAKEDEATH)))
 		appears_dead = TRUE
 
-		if(isliving(user) && HAS_TRAIT(user, TRAIT_NAIVE))
+		var/obj/item/clothing/glasses/G = get_item_by_slot(ITEM_SLOT_EYES)
+		var/are_we_in_weekend_at_bernies = G && G.tint && buckled && istype(buckled, /obj/vehicle/ridden/wheelchair)

```

Comment:
```suggestion
		var/are_we_in_weekend_at_bernies = G?.tint && buckled && istype(buckled, /obj/vehicle/ridden/wheelchair)
```

Reply:
oh my god, this is a beautiful new thing to learn